### PR TITLE
Alfredapi 390 antisymlink

### PR DESCRIPTION
**Pull request by WimCrols**
Fixes https://xenitsupport.jira.com/browse/ALFREDAPI-390

- [x] Is [CHANGELOG.md](https://github.com/xenit-eu/alfred-api/blob/master/CHANGELOG.md) extended?
- [x] Does this PR avoid breaking the API? 
    Breaking changes include adding, changing or removing endpoints and/or JSON objects used in requests and responses.
- [x] Does the PR comply to REST HTTP result codes policy outlined in the [developer guide](https://github.com/xenit-eu/alfred-api/blob/master/developer-documentation)?
- [x] Does the PR follow our [coding styleguide and other active procedures](https://xenitsupport.jira.com/wiki/spaces/XEN/pages/624558081/XeniT+Enhancement+Proposals+XEP)?
- [x] Is usage of `this.` prefix avoided?

See [README.md](./README.md) for full explanation.

### CODE REVIEW

File: apix-impl/build.gradle [hunk 1]
```diff
@@ -27,11 +45,11 @@ subprojects {
     artifacts {
         ampArtifact amp
     }
-
+    
     sourceSets {
         main {
             java {
-                srcDirs = ['src/main/java', 'src/main/java-shared']
+                srcDirs = ['src/main/java', '../src/main/java']
```
Comment (kpiot123) [inline apix-impl/build.gradle:52]: @WimCrols I think it is more clear to change `'00/src/main/java'` to `"${project(":apix-integrationtests").path}/src/main/java"`. What do you think?

File: apix-impl/build.gradle [hunk 2]
```diff
@@ -40,7 +58,7 @@ subprojects {
         }
         test {
             java {
-                srcDirs = ['src/test/java', 'src/test/java-shared']
+                srcDirs = ['../src/test/java']
```
Comment (kpiot123) [inline apix-impl/build.gradle:61]: @WimCrols Same here as my previous comment.

File: apix-integrationtests/build.gradle [hunk 3]
```diff
@@ -13,12 +13,12 @@ subprojects {
...
         main {
             java {
-                srcDirs = ['src/integration-test/java-shared', 'src/main/java']
+                srcDir '../../apix-impl/src/integration-test'
                 // This is needed because client side, these dependencies are not really provided
                 runtimeClasspath += configurations.compileOnly
             }
             resources {
-                srcDir "${project.parent.projectDir}/src/integration-test/resources"
+                srcDir '../src/integration-test/resources'
```
Comment (kpiot123) [inline apix-integrationtests/build.gradle:21]: @WimCrols Same here as my previous comments but with `"${project(":apix-integrationtests").path}/src/main/java"`

File: apix-integrationtests/build.gradle [hunk 4]
```diff
@@ -13,12 +13,12 @@ subprojects {
     sourceSets {
         main {
             java {
-                srcDirs = ['src/integration-test/java-shared', 'src/main/java']
+                srcDir '../../apix-impl/src/integration-test'
```
Comment (kpiot123) [inline apix-integrationtests/build.gradle:16]: @WimCrols Same here as my previous comments

File: apix-impl/build.gradle (hunk 1 above)
Comment (WimCrols) [reply apix-impl/build.gradle:52]: I considered it but think the ../ is actually more readable.

File: apix-integrationtests/build.gradle (hunk 4 above)
Comment (WimCrols) [reply apix-integrationtests/build.gradle:16]: Well.. I changed it for consistency, but of courese if we change it in one spot, we should do all.

File: apix-impl/build.gradle (hunk 1 above)
Comment (kpiot123) [reply apix-impl/build.gradle:52]: That is also ok for me. It is probably a matter of opinion. I will approve the changes.

**Review by kpiot123 — APPROVED** [review]

**Pull request merged**